homeassistant/components/geniushub/__init__.py

```diff
@@ -47,7 +49,9 @@
 
 SCAN_INTERVAL = timedelta(seconds=60)
 
-_V1_API_SCHEMA = vol.Schema({vol.Required(CONF_TOKEN): cv.string})
+_V1_API_SCHEMA = vol.Schema(
```

Comment:
Don't name the constants with underscore prefix. It's not needed.

Reply:
Done.